docker/docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+version: "3.6"
+
+services:
+  jekyll:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    ports:
+      - 4000:4000
+    volumes:
+      - ..:/usr/src/app
+    # If at runtime you need to pass a proxy server, make use of this
+    # gitignored file.
+    env_file:
+      - .env
```

Comment:
I think these lines need to be commented out given that by default the user won't have any such file. Otherwise this will give an error:
$ docker-compose -f docker/docker-compose.yaml up --build
ERROR: Couldn't find env file: /Users/lehors/Projects/github.com/hyperledger/tsc/docker/.env

Reply:
Good catch, thanks. I removed this section. `docker` CLI has default behavior of reading `.env` file from the work directory. This allows one to inject additional attributes without explicitly having a section for it in the `docker-compose.yaml`.